aea/aea_builder.py

```diff
@@ -875,6 +921,14 @@ def _set_from_configuration(
         self.set_timeout(agent_configuration.timeout)
         self.set_execution_timeout(agent_configuration.execution_timeout)
         self.set_max_reactions(agent_configuration.max_reactions)
+        if agent_configuration.decision_maker_handler != {}:
+            dotted_path_full = (
+                agent_configuration.decision_maker_handler["dotted_path"]
+                + ":"
+                + agent_configuration.decision_maker_handler["class_name"]
+            )
+            file_path = agent_configuration.decision_maker_handler["file_path"]
+            self.set_decision_maker_handler(dotted_path_full, file_path)
```

Comment:
can be cleaned up when we got it to work

Reply:
I think we could use the `dotted_path:ClassName` format in the configuration file, rather than building it in the code.

Also, worth exploring the `pytest` approach: `path/to/python/module.py::SomeClassName`